leaflet-image.js

```diff
@@ -30,6 +30,7 @@ module.exports = function leafletImage(map, callback) {
     // layers are drawn in the same order as they are composed in the DOM:
     // tiles, paths, and then markers
     map.eachLayer(drawTileLayer);
+    map.eachLayer(drawEsriDynamicLayer);
```

Comment:
Sorry. 
I have moved changes to index.js and also have added check for L.esri is defined.